### DISCUSSION TITLE
github page general problem

### DISCUSSION THREAD

**Discussion by ori1992** [community/community#101817] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

hey, 
I am trying to write code for the website like add a link but instead of a link I see literally what I write
i wrote this code and got exactly this written, what is the problem? 
![image](https://github.com/community/community/assets/157274918/0a63913b-45cd-4311-a15e-50692b868303)

**Comment by AidenJamesYt** [comment]
Did you put a space in between the brackets and the parenthesis?

**Reply by AidenJamesYt** [reply]
If you did, you dont want a space in between. You can check out more about relative links [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links). 

Hope I could help! :D
